std/math.d

```diff
@@ -7341,6 +7341,8 @@ private T powIntegralImpl(PowType type, T)(T val)
 
     if (x == 0)
         return x;
+    if (x == Unqual!T.max || x == Unqual!T.min)
```

Comment:
Thanks for the quick action.

I think the condition must only apply to PowType.ceil and should distinguish between signed and unsigned:
```
static if (type == PowType.ceil)
{
    if(x > T.max / 2 || x == T.min)
        return 0;
}
```